packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -303,25 +305,53 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ patientUuid, showAddBu
     [orderTypes, t],
   );
 
+  const handleDateFilterChange = ([startDate, endDate]) => {
+    if (startDate) {
+      setSelectedFromDate(startDate.toISOString());
+      if (selectedToDate && startDate && selectedToDate < startDate) {
+        setSelectedToDate(startDate.toISOString());
+      }
+    }
+    if (endDate) {
+      setSelectedToDate(endDate.toISOString());
+      if (selectedFromDate && endDate && selectedFromDate > endDate) {
+        setSelectedFromDate(endDate.toISOString());
+      }
+    }
+  };
   return (
     <>
-      <div className={styles.dropdownContainer}>
-        <Dropdown
-          id="orderTypeDropdown"
-          items={orderTypesToDisplay}
-          itemToString={(orderType: OrderType) => (orderType ? capitalize(orderType.display) : '')}
-          label={t('allOrders', 'All orders')}
-          onChange={(e: { selectedItem: Order }) => {
-            if (e.selectedItem.display === 'All') {
-              setSelectedOrderTypeUuid(null);
-              return;
-            }
-            setSelectedOrderTypeUuid(e.selectedItem.uuid);
+      <div className={styles.filterContainer}>
+        <div className={styles.dropdownContainer}>
+          <Dropdown
+            id="orderTypeDropdown"
+            items={orderTypesToDisplay}
+            itemToString={(orderType: OrderType) => (orderType ? capitalize(orderType.display) : '')}
+            label={t('allOrders', 'All orders')}
+            onChange={(e: { selectedItem: Order }) => {
+              if (e.selectedItem.display === 'All') {
+                setSelectedOrderTypeUuid(null);
+                return;
+              }
+              setSelectedOrderTypeUuid(e.selectedItem.uuid);
+            }}
+            selectedItem={orderTypes?.find((x) => x.uuid === selectedOrderTypeUuid)}
+            titleText={t('selectOrderType', 'Select order type') + ':'}
+            type="inline"
+          />
+        </div>
+        <p className={styles.helperText}>{t('dateRange', 'Date range')} :</p>
```

Comment:
```suggestion
        <span className={styles.rangeLabel}>{t('dateRange', 'Date range')} :</span>
```

`<span>` or some other inline element is more ideal here as the element doesn't span the full width (because of the range datepickers). Also, this element will need some horizontal margin to the datepickers, so we could do something like:

		.rangeLabel {
		  @extend .helperText;
		  margin-right: layout.$spacing-02;
		}